src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -410,4 +408,22 @@ public void testVCFWithNoRecords() {
         Assert.assertTrue(output.length() > 0);
     }
 
+    @Test
+    public void testEnsemblGtfIndex() {
```

Comment:
It would be good to check that the using the index to look up records gets you the ones you wanted as part of this test.

Reply:
Sounds good.  I've added a couple parts to the test that does this.